services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerConfig.java

```diff
@@ -486,7 +489,10 @@ public VeniceControllerConfig(VeniceProperties props) {
     this.daVinciPushStatusScanThreadNumber = props.getInt(DAVINCI_PUSH_STATUS_SCAN_THREAD_NUMBER, 4);
     this.daVinciPushStatusScanNoReportRetryMaxAttempt =
         props.getInt(DAVINCI_PUSH_STATUS_SCAN_NO_REPORT_RETRY_MAX_ATTEMPTS, 6);
-    this.daVinciPushStatusScanMaxOfflineInstance = props.getInt(DAVINCI_PUSH_STATUS_SCAN_MAX_OFFLINE_INSTANCE, 10);
+    this.daVinciPushStatusScanMaxOfflineInstanceCount =
+        props.getInt(DAVINCI_PUSH_STATUS_SCAN_MAX_OFFLINE_INSTANCE_COUNT, 10);
+    this.daVinciPushStatusScanMaxOfflineInstanceRatio =
+        props.getDouble(DAVINCI_PUSH_STATUS_SCAN_MAX_OFFLINE_INSTANCE_RATIO, 0.1d);
```

Comment:
This is 10%

Reply:
Will 10% be too high as the default?
I was thinking something like 1% or 2%..

Reply:
But in the previous discussion, you mentioned in large scale deployment, it can bring down a lot of instance at the same time, is that right? For example, if an app cluster has 1k host, 10% is 100, and 1% is 10. Will the parallel deployment take down 10+ at once?